build.gradle

```diff
@@ -20,34 +20,36 @@ apply plugin: 'org.grails.grails-plugin'
 apply plugin: 'org.grails.internal.grails-plugin-publish'
 apply plugin: 'maven-publish'
 
-sourceCompatibility = 11
-targetCompatibility = 11
-
 repositories {
-    maven { url "https://repo.grails.org/grails/core" }
+    maven { url = 'https://repo.grails.org/grails/core' }
 }
 
 configurations {
```

Comment:
Is this block doing anything now?

Reply:
Yes, it is registering a configuration named `documentation`. I'll rewrite it so it becomes more clear.